napari/settings/_base.py

```diff
@@ -49,6 +49,10 @@ class EventedSettings(BaseSettings, EventedModel):
     """
 
     # provide config_path=None to prevent reading from disk.
```

Comment:
Is this comment still relevant or should it be updated for the newly added `Config`?

Reply:
yes. The change above only explicitly enforces which pydantic.BaseModel. Config is used. It is not config for settings.